Integrations/GoogleVisionAPI/GoogleVisionAPI.yml

```diff
@@ -0,0 +1,46 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Google Vision AI
+  version: -1
+configuration:
+- display: Google service account JSON (a credentials JSON generated from Google API
+    Manager or from GCP console)
+  name: auth_json
+  required: true
+  type: 4
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: Image processing with Google Vision API
+display: Google Vision AI
+name: Google Vision AI
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The entry ID of the image to be processed
+      isArray: true
+      name: entry_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Detects brand logos in the given image
+    execution: false
+    name: google-vision-detect-logos
+    outputs:
+    - contextPath: GoogleVisionAPI
```

Comment:
I think you are not handling it correctly in the code, let's go over this together in the execution review.